pokemap.py

```diff
@@ -5,23 +5,25 @@
 import struct
 import sys
 import argparse
+import os
 
 DEBUG_MODE = False
 def debug(*args, **kwargs):
   if DEBUG_MODE:
     print(*args, **kwargs)
 
 def main():
+
   parser = argparse.ArgumentParser(description="do a wee bit o' data rippin from a rom")
   parser.add_argument("-v", "--verbose",
                       help="Print information while running to help debug", action="store_true",
                       dest="verbose")
   parser.add_argument('rom_file', metavar='rom', type=str,
                        help='a fire red rom')
-  parser.add_argument("-e","--extension",
-                      help="Specify the output file extension",
-                      choices=['png','jpeg','tga','bmp'],
-                      default='bmp')
+  parser.add_argument("-o","--outfile",
+                      help="Specify the output file name/extension",
+                      action=CheckExt({'png','bmp','jpeg','tga'}),
+                      default='wholemap.bmp')
```

Comment:
This serves the same purpose, but is just more robust and allows for changing name+ext